packages/terra-application-header-layout/src/ApplicationHeaderLayout.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+
+import styles from './ApplicationHeaderLayout.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Extensions element to be placed before the end of the header.
+   * */
+  extensions: PropTypes.element,
```

Comment:
Super nitpicky - because each of these are singular elements, wouldn't it make more sense for these props to be in singular form, i.e. `extension` and `utility`?

Reply:
Makes sense enough, but these were the names reviewed by UX. It will take a single component, but that component will have multiple components.